ui/app/src/utils/string.ts

```diff
@@ -67,6 +67,10 @@ export function isJSON(str: string): boolean {
   throw new Error('[isJSON] Not implemented.');
 }
 
+export function isCamelCase(str: string) {
```

Comment:
This doesn't test for camel case.
```suggestion
export function hasUppercaseChars(str: string) {
```